src/iuse_actor.cpp

```diff
@@ -4518,3 +4519,36 @@ std::unique_ptr<iuse_actor> change_scent_iuse::clone() const
 {
     return std::make_unique<change_scent_iuse>( *this );
 }
+
+std::unique_ptr<iuse_actor> effect_on_conditons_actor::clone() const
+{
+    return std::make_unique<effect_on_conditons_actor>( *this );
+}
+
+void effect_on_conditons_actor::load( const JsonObject &obj )
+{
+    description = obj.get_string( "description" );
+    for( const std::string &eoc : obj.get_string_array( "effect_on_conditions" ) ) {
+        eocs.push_back( effect_on_condition_id( eoc ) );
+    }
+}
+
+void effect_on_conditons_actor::info( const item &, std::vector<iteminfo> &dump ) const
+{
+    dump.emplace_back( "DESCRIPTION", description );
+}
+
+cata::optional<int> effect_on_conditons_actor::use( player &, item &it, bool,
```

Comment:
`, bool,` ?

Reply:
Good catch!  I've just made the code better by using the player ref which should fix item usage for npcs.  The bool on the other hand is something to do with items with time outs or something with activation.  I don't fully understand it, but its not needed in this case and is similarly ignored in a lot of item use actors.